globus_automate_client/cli/flows.py

```diff
@@ -383,6 +396,17 @@ def flow_update(
         ("--administered-by", administered_by or None),
     )
 
+    # Special cases:
+    # * If the user specifies a single empty string, replace [""] with []
+    #   so all values currently set on the flow will be erased.
+    # * If the user specifies nothing, replace the default empty list with None
+    #   to prevent erasure of the values currently set on the flow.
+    flow_viewer = [] if flow_viewer == [""] else (flow_viewer or None)
+    flow_starter = [] if flow_starter == [""] else (flow_starter or None)
+    flow_administrator = (
+        [] if flow_administrator == [""] else (flow_administrator or None)
+    )
```

Comment:
I think it'd be clearer to add a custom flag mutexed with these keys instead of a special value.

Something like
`globus-automate update-flow --no-flow-viewers --no-flow-administrators --flow-starter urn:...`

Reply:
Run permissions are handled in the same way (the core logic of this code was copied verbatim from the `update_run()` function).

It will take some additional work to introduce that mutex idea, but it can be done. My gut is to follow what exists for run permissions, but let me know if you have a strong desire to do it this way.